pages/ksphere/kommander/1.1.0-beta/tutorials/configure-rbac/index.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: layout.pug
+navigationTitle: Granting access to Kubernetes resources in Kommander
+title: Granting access to Kubernetes resources in Kommander
+menuWeight: 1
+excerpt: Configure role-based access control for Kubernetes resources in Kommander
+---
+
+## Before you begin
+
+This procedure requires the following configurations and background: 
+- A Konvoy cluster with [Kommander installed](https://docs.d2iq.com/ksphere/kommander/1.1.0-beta/install/).
+- An [Identity Provider](https://docs.d2iq.com/ksphere/kommander/1.1.0-beta/tutorials/authorize-all-users/) has been created. 
+- Some familiarity with [Kubernetes role-based access control](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) principles. 
+
+## Create Role-Based Access Control Personas
+
+Manage access to resources on Kubernetes clusters through the [RBAC API](https://kubernetes.io/docs/reference/access-authn-authz/rbac/). The base premise revolves around assigning users to groups, which are then associated with roles that control their level of access through role bindings.
+This procedure describes how a cluster administrator can assign different levels of access to Kubernetes and Kommander resources in the Kommander UI.
+
+Kommander has [three levels of access control](https://docs.d2iq.com/ksphere/kommander/1.1.0-beta/operations/access-control/) to support global, workspace, and project use cases. Kommander creates default roles at all three levels to simplify the process of assigning users to different levels of access to resources.
+
+In this procedure, using these default roles, you will create four different personas with users assigned to them:
+- *Global admin*: these users can administer all Kubernetes and Kommander resources on the management cluster and all target clusters.
+- *Global viewer*: these users have read-only viewing access to all Kubernetes and Kommander resources on the management cluster and all target clusters.
+- *Workspace admin*: these users can administer the workspaces they have specifically been given access to.
+- *Workspace viewer*: these users can access the workspaces they have specifically been given access to in a readonly capacity.
+
+In the Kommander UI, do the following:
+1. Select *Global* in the header drop-down. 
+2. Select *Administration* > *Identity Providers*.
+3. Select the *Group* tab. 
+4. Create a group to represent each of the 4 personas:
```

Comment:
I find this part a bit misleading because these seem to reflect the role names a bit too closely. I wonder if we could simply make it a prerequisite to set up virtual groups or just choose names that may align more closely to organization roles-- like managers, contractors, etc

Reply:
Yeah, I think a prereq with a link to groups would be better. 
If you haven't done so already, create a group to represent each of the personas. For more details on creating groups, see _<link to groups topic>_.